### ISSUE TITLE
Group layer toggle-off does not turn layers off as expected

### ISSUE THREAD

**Issue by josephmsms** [ggcity/map-viewer#4]

**Comment by mooman** [comment]
What is a use case for turning off a whole group of layers?

**Comment by mooman** [comment]
I agree the toggle should not turn off while the layer stays in, however.

**Comment by josephmsms** [comment]
Or get rid of toggle

**Comment by mooman** [comment]
Toggle needed when we have multiple groups. Like radio buttons on forms. Maybe have toggle that does nothing when on already.
